torchbraid/torchbraid_callbacks.pyx

```diff
@@ -82,21 +83,35 @@ cdef int my_step(braid_App app, braid_Vector ustop, braid_Vector fstop, braid_Ve
 
       tstart = 0.0
       tstop = 5.0
+      tindex = -1
       level = -1
       braid_StepStatusGetTstartTstop(status, &tstart, &tstop)
       braid_StepStatusGetLevel(status, &level)
       braid_StepStatusGetDone(status, &done)
+      braid_StepStatusGetDone(status, &done)
+      braid_StepStatusGetTIndex(status, &tindex)
```

Comment:
When called on coarser grids, this tindex is the index of the time-point on that coarse level, hence needs to be converted to the corresponding fine-grid time-index (e.g. multiply by the "(cfactor)^level" I guess)